<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OutputsWidget</class>
 <widget class="QWidget" name="OutputsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>662</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Output directory: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="OutputDirLabel">
       <property name="styleSheet">
        <string notr="true">font:bold;</string>
       </property>
       <property name="text">
        <string notr="true">OutputDirLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ExploreButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Explore</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ClearButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Clear contents</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="OutputDirView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
